lib/pf/scan.pm

```diff
@@ -334,8 +341,8 @@ Check if the category matches the configuration of the scanner
 sub matchCategory {
     my ($self, $node_attributes) = @_;
     my $category = [split(/\s*,\s*/, $self->{_categories})];
-    my $node_cat = $node_attributes->{'category'};
-
+    my $node_cat = $node_attributes->{'category'} || 'default';
```

Comment:
No role *is not* the default role. These are 2 completely different things in the PacketFence logic currently

Reply:
Ok good to know , check my changes because we still have to return before "any { $_ eq $node_cat } @$category" as it will cause an error when $node_cat isn't defined